owncloudComLibrary/src/test/responses/com.owncloud.android.lib.resources.webfinger.responses/not_containing_releavant_info_response.json

```diff
@@ -0,0 +1,10 @@
+{
+  "subject": "acct:peter.sine@gurken.xxx",
+  "links" :
+  [
+    {
+      "rel" : "http://webfinger.example/rel/businesscard",
+      "href" : "https://www.example.com/~bob/bob.vcf"
+    }
+  ]
+}
```

Comment:
Add final blank line. Also, I think the name of this file doesn't fit with the name that is used in the tests. In this case, it is called `..._releavant_...`, it should be "relevant", is that correct?